<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QMainWindow" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btnAddCustomer">
      <property name="text">
       <string>Add karbar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnRemoveCustomer">
      <property name="text">
       <string>Remove karbar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEditProfile">
      <property name="text">
       <string>Edit Profile</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnViewAllCustomers">
      <property name="text">
       <string>View All karbar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnViewAllAccounts">
      <property name="text">
       <string>View All Accounts</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnExit">
      <property name="text">
       <string>exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>559</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
